tzkt/fetchTransactionTree.ts

```diff
@@ -46,85 +52,98 @@ async function fetchTransactions(
   return results;
 }
 
+function isUserWallet(wallet: Wallet) {
+  const address = wallet.address;
+  const isUserWallet = !(
+    wallet.address.startsWith("KT1") ||
+    exchangeAliases.some((alias) => alias === wallet.alias)
```

Comment:
juste une suggestion, c'est cool comme ca aussi
```suggestion
    exchangeAliases.includes(wallet.alias)
```